cclib/parser/molcasparser.py

```diff
@@ -636,10 +675,19 @@ def extract(self, inputfile, line):
         #         60 H20   1s      0.1835
         #  --
         if '++    Molecular orbitals:' in line:
+
             self.skip_lines(inputfile, ['d', 'b'])
             line = next(inputfile)
-            if 'Natural orbitals' not in line:
-                self.skip_lines(inputfile, ['b', 'symm'])
+
+            # We don't currently support parinsg natural orbitals or active space orbitals.
```

Comment:
`parinsg -> parsing`

Reply:
Fixed in a0671265b22f7a1e8afb76d929405398f3fc1499